cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/store/AppMetadataStoreTest.java

```diff
@@ -279,16 +279,19 @@ public void testInvalidStatusPersistence() throws Exception {
     });
     final RunId runId6 = RunIds.generate(runIdTime.incrementAndGet());
     final ProgramRunId programRunId6 = program.run(runId6);
+    Long currentTime = System.currentTimeMillis();
     // STARTING status will be ignored if there's any existing record
     txnl.execute(() -> {
       recordProvisionAndStart(programRunId6, metadataStoreDataset);
       metadataStoreDataset.recordProgramSuspend(programRunId6,
-                                                AppFabricTestHelper.createSourceId(sourceId.incrementAndGet()));
+                                                AppFabricTestHelper.createSourceId(sourceId.incrementAndGet()),
+                                                currentTime);
```

Comment:
even thought the time unit doesn't matter here, but should be in seconds

Reply:
done